README.md

```diff
@@ -1,33 +1,89 @@
+   * [Test Suite](#test-suite)
+* [Support and documentation](#support-and-documentation)
+
+### Prerequisites
+* Unity Editor version 2018.*
+* Android 4.1 (API level 16)+ (when building for Android)
+* iOS 9.0+ (when building for iOS)
+
+## Project Setup
+
+### Importing FairBid Unity Plugin
+
+Before building the project, you need to import the FaidBid Unity Plugin. For this, you need to download FairBid Unity Plugin from [Fyber's dev portal](https://dev-unity.fyber.com/docs/integration).
+
+With this project open in Unity, double-click the downloaded FairBid Unity Plugin package (FairBidSDK.unitypackage) file to import the FairBid Unity Plugin. When the Import Unity Package window opens, click All to select everything before importing.
+
+#### Set the target platform in Build Settings
+
+After the import is completed. You need to select the target platform (Android/iOS) you wish to run this project on and click on `File -> Build Settings`.
+
+#### Platform specific setup
+
+- [Android specific setups instructions](https://dev-unity.fyber.com/docs/unity-sdk-integration-new-draft#android-specific-setup)
+- [iOS specific setups instructions](https://dev-unity.fyber.com/docs/unity-sdk-integration-new-draft#ios-specific-setup)
+
+## Build and Run
+
+You're now ready to run this project. Click `File -> Build and Run`. Unity will build the project and run the *Sample Unity application* in the targeted platform (Android/iOS).
+
+## Navigating the project code
+
+In the folder `Assets/Scenes/`, we have the following scenes:
+* [MainScene](https://github.com/Heyzap/fairbid-sample-app-unity/blob/master/Assets/Scenes/MainScene.cs) - SDK is Initialized
+
+* [BannerScene](https://github.com/Heyzap/fairbid-sample-app-unity/blob/master/Assets/Scenes/BannerScene.cs) - Requesting Banner Ads
+  
+* [InterstitialScene](https://github.com/Heyzap/fairbid-sample-app-unity/blob/master/Assets/Scenes/InterstitialScene.cs) - Requesting Interstitial Ads
+
+* [RewardedScene](https://github.com/Heyzap/fairbid-sample-app-unity/blob/master/Assets/Scenes/RewardedScene.cs) - Requesting Rewarded Ads
+
+## Mediation
+
+If you want to test mediation features using this Sample app, there are a couple of steps you will need to take first: 
+
+1. Add the 3rd party mediation network configuration from their dashboard to the Fyber's console. Refer to the section [Integrating Mediation Partners
+](https://fyber-mediation.fyber.com/docs/integrating-mediation) in Fyber's dev portal for instructions.
+2. Integrate 3rd party networks in the project. Go to the [mediation network table](https://fyber-mediation.fyber.com/docs/supported-networks#platform-unity) and select the network you wish to integrate in the sample app. Then follow the instructions respective to the integration approach you choose. There are two different approaches to achieve this:.
+
+   - [Using External Dependency Manager for Unity](https://fyber-mediation.fyber.com/docs/supported-networks#platform-unity)
```

Comment:
We can now update these links

Reply:
I updated them in this commit

a824ddd